zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterFactory.java

```diff
@@ -1094,14 +1075,15 @@ private String getInterpreterClassFromInterpreterSetting(InterpreterSetting sett
     return null;
   }
 
-  private Interpreter getInterpreter(String noteId, InterpreterSetting setting, String name) {
+  private Interpreter getInterpreter(String noteId, InterpreterSetting setting, String name,
+                                     String userName) {
```

Comment:
Probably a nitpick, but horizontal alignment is controversial idea and generally [is discouraged by the styleguide](https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment) as it creates a "blast radius" of re-formatting in case of future changes i.e renaming a function.